lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -359,6 +359,24 @@ def _configure_nodes(self, environment: Environment, log: Logger) -> None:
                             yaml.safe_load(file)
                         )
 
+            # Configure data disks.
+            if node_space.disk:
+                assert isinstance(
+                    node_space.disk.data_disk_count, int
+                ), f"actual: {type(node_space.disk.data_disk_count)}"
+                assert isinstance(
+                    node_space.disk.data_disk_size, int
+                ), f"actual: {type(node_space.disk.data_disk_size)}"
+
+                for i in range(node_space.disk.data_disk_count):
+                    data_disk = DataDiskContext()
+                    data_disk.file_path = os.path.join(
+                        vm_disks_dir, f"{node_context.vm_name}-data-{i}.qcow2"
```

Comment:
It needs to make sure the vm_name is global unique. As my understand, it's a random string, which across environments and test runs. It still has a chance to conflict. It's better to create a folder for each environment and use the folder to detect conflicts.

Reply:
Each VM is already given their own directory.

Reply:
Also, this is just following the existing logic used by the QEMU orchestrator.